keylime/tpm/tpm_main.py

```diff
@@ -434,15 +435,15 @@ def __get_pub_ek(self) -> None:  # assumes that owner_pw is correct at this poin
 
     def __create_aik(
         self, asym_alg: Optional[str] = None, hash_alg: Optional[str] = None, sign_alg: Optional[str] = None
-    ):
+    ) -> None:
         if hash_alg is None:
             hash_alg = self.defaults["hash"]
         if asym_alg is None:
             asym_alg = self.defaults["encrypt"]
         if sign_alg is None:
             sign_alg = self.defaults["sign"]
 
-        owner_pw = self.get_tpm_metadata("owner_pw")
+        owner_pw = cast(str, self.get_tpm_metadata("owner_pw"))
```

Comment:
`owner_pw = str(self.get_tpm_metadata("owner_pw"))`?

Reply:
This also works. Is this how we want to generally do it rather than using casts?

Reply:
The issue is that a `cast` does not change the type. I guess that what we want is a type-safe Python, and a `cast` does not help to achieve this goal. It is only a way to bypass the type checker[1]

[1] https://docs.python.org/3/library/typing.html#typing.cast